dev/tasks/tasks.yml

```diff
@@ -214,6 +214,73 @@ groups:
     - test-conda-python-3.8-jpype
     # - test-conda-cpp-hiveserver2
 
+  verify-rc:
+    - verify-rc-binaries
+    - verify-rc-wheels-linux
+    - verify-rc-wheels-macos
+    - verify-rc-wheels-windows
+    - verify-rc-source-macos-java
+    - verify-rc-source-macos-csharp
+    - verify-rc-source-macos-ruby
+    - verify-rc-source-macos-python
+    - verify-rc-source-macos-js
+    - verify-rc-source-macos-go
+    - verify-rc-source-macos-rust
+    - verify-rc-source-macos-integration
+    - verify-rc-source-linux-java
+    - verify-rc-source-linux-csharp
+    - verify-rc-source-linux-ruby
+    - verify-rc-source-linux-python
+    - verify-rc-source-linux-js
+    - verify-rc-source-linux-go
+    - verify-rc-source-linux-rust
+    - verify-rc-source-linux-integration
+    - verify-rc-source-windows
+
+  verify-rc-wheels:
+    - verify-rc-wheels-linux
+    - verify-rc-wheels-macos
+    - verify-rc-wheels-windows
+
+  verify-rc-source:
+    - verify-rc-source-macos-java
+    - verify-rc-source-macos-csharp
+    - verify-rc-source-macos-ruby
+    - verify-rc-source-macos-python
+    - verify-rc-source-macos-js
+    - verify-rc-source-macos-go
+    - verify-rc-source-macos-rust
+    - verify-rc-source-macos-integration
+    - verify-rc-source-linux-java
+    - verify-rc-source-linux-csharp
+    - verify-rc-source-linux-ruby
+    - verify-rc-source-linux-python
+    - verify-rc-source-linux-js
+    - verify-rc-source-linux-go
+    - verify-rc-source-linux-rust
+    - verify-rc-source-linux-integration
+    - verify-rc-source-windows
+
+  verify-rc-source-macos:
+    - verify-rc-source-macos-java
+    - verify-rc-source-macos-csharp
+    - verify-rc-source-macos-ruby
+    - verify-rc-source-macos-python
+    - verify-rc-source-macos-js
+    - verify-rc-source-macos-go
+    - verify-rc-source-macos-rust
+    - verify-rc-source-macos-integration
+
+  verify-rc-source-linux:
+    - verify-rc-source-linux-java
+    - verify-rc-source-linux-csharp
+    - verify-rc-source-linux-ruby
+    - verify-rc-source-linux-python
+    - verify-rc-source-linux-js
+    - verify-rc-source-linux-go
+    - verify-rc-source-linux-rust
+    - verify-rc-source-linux-integration
```

Comment:
Maybe https://issues.apache.org/jira/browse/ARROW-8356 could help?

Reply:
Yeah, I've wanted something like that before too, just haven't wanted to wade into (untested) crossbow code to do it yet.

Reply:
Good idea, done https://github.com/apache/arrow/pull/6866